### PR TITLE
add terraform-ls 0.31.2

### PR DESCRIPTION
**Pull request by Slach**
terraform-ls
Terraform LSP Server

### CODE REVIEW

File: bucket/terraform-ls.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "0.31.2",
+    "description": "Terraform Language Server",
+    "url":"https://github.com/hashicorp/terraform-ls",
```
Comment (StarsbySea) [inline bucket/terraform-ls.json:4]:
```suggestion

```

**Review by StarsbySea — CHANGES_REQUESTED** [review]

**Comment by HUMORCE** [comment]
main bucket.

**Comment by Slach** [comment]
create separate PR in main bucket https://github.com/ScoopInstaller/Main/pull/4887

**Pull request closed without merging**